<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Збут</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ico.qrc">
    <normaloff>:/ico/fork3256x256.ico</normaloff>:/ico/fork3256x256.ico</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="maximumSize">
         <size>
          <width>58</width>
          <height>48</height>
         </size>
        </property>
        <property name="icon">
         <iconset resource="ico.qrc">
          <normaloff>:/ico/Button Add.ico</normaloff>:/ico/Button Add.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>58</width>
          <height>48</height>
         </size>
        </property>
        <property name="icon">
         <iconset resource="ico.qrc">
          <normaloff>:/ico/Button Delete.ico</normaloff>:/ico/Button Delete.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <property name="autoRepeatDelay">
         <number>0</number>
        </property>
        <property name="autoRepeatInterval">
         <number>0</number>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>58</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="ico.qrc">
          <normaloff>:/ico/edit.ico</normaloff>:/ico/edit.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_11">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>58</width>
          <height>48</height>
         </size>
        </property>
        <property name="icon">
         <iconset resource="ico.qrc">
          <normaloff>:/ico/kghostview_3262.ico</normaloff>:/ico/kghostview_3262.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>58</width>
          <height>48</height>
         </size>
        </property>
        <property name="icon">
         <iconset resource="ico.qrc">
          <normaloff>:/ico/print.ico</normaloff>:/ico/print.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>93</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maxVisibleItems">
         <number>12</number>
        </property>
        <item>
         <property name="text">
          <string>Січень</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Лютий</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Березень</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Квітень</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Травень</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Червень</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Липень</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Серпень</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Вересень</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Жовтень</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Листопад</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Грудень</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="minimum">
         <number>1990</number>
        </property>
        <property name="maximum">
         <number>2099</number>
        </property>
        <property name="value">
         <number>2012</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>49</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ico.qrc">:/ico/search.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>25</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::WheelFocus</enum>
        </property>
        <property name="cursorPosition">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>140</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>140</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="title">
         <string>РЕЄСТР НАКЛАДНИХ</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <widget class="QPushButton" name="pushButton_6">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>5</x>
           <y>85</y>
           <width>181</width>
           <height>41</height>
          </rect>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Прихід</string>
         </property>
         <property name="icon">
          <iconset resource="ico.qrc">
           <normaloff>:/ico/arrowup.ico</normaloff>:/ico/arrowup.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_10">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>5</x>
           <y>30</y>
           <width>180</width>
           <height>41</height>
          </rect>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Збут</string>
         </property>
         <property name="icon">
          <iconset resource="ico.qrc">
           <normaloff>:/ico/arrowdown.ico</normaloff>:/ico/arrowdown.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="title">
         <string>ДОВІДКОВА СИСТЕМА</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <widget class="QPushButton" name="pushButton_7">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>5</x>
           <y>70</y>
           <width>181</width>
           <height>41</height>
          </rect>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Замовники</string>
         </property>
         <property name="icon">
          <iconset resource="ico.qrc">
           <normaloff>:/ico/contacts.ico</normaloff>:/ico/contacts.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_8">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>5</x>
           <y>20</y>
           <width>181</width>
           <height>40</height>
          </rect>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Вироби</string>
         </property>
         <property name="icon">
          <iconset resource="ico.qrc">
           <normaloff>:/ico/dolly.ico</normaloff>:/ico/dolly.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QTableView" name="tableView">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::CurrentChanged|QAbstractItemView::SelectedClicked</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Довідкова система</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Сервис</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Отчеты</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>Справка</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_6"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/dolly.ico</normaloff>:/ico/dolly.ico</iconset>
   </property>
   <property name="text">
    <string>Продукція</string>
   </property>
  </action>
  <action name="action_2">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/contacts.ico</normaloff>:/ico/contacts.ico</iconset>
   </property>
   <property name="text">
    <string>Замовники</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>pushButton_10</tabstop>
  <tabstop>pushButton_8</tabstop>
  <tabstop>pushButton_7</tabstop>
  <tabstop>lineEdit</tabstop>
 </tabstops>
 <resources>
  <include location="ico.qrc"/>
 </resources>
 <connections/>
</ui>
